src/main/java/strategy/OperationBalance.java

```diff
@@ -0,0 +1,18 @@
+package strategy;
+
+import db.Storage;
+import model.Transaction;
+
+public class OperationBalance implements OperationStrategy {
+
+    private Transaction transaction;
+
+    public OperationBalance(Transaction transaction) {
+        this.transaction = transaction;
+    }
+
+    @Override
+    public void handleOperation(Storage storage) {
```

Comment:
Use the constructor to set the Services, Repositories/DBs
and The methods should basically accept the dtos, models
fix it everywhere
```suggestion
    private Storage storage;

    public OperationBalance(Storage storage) {
        this.storage = storage;
    }

    @Override
    public void handleOperation(Transaction transaction) {
```